tests/unit/plugins/test_python_plugin.py

```diff
@@ -193,3 +197,94 @@ def test_call_should_remove_symlinks(plugin, new_dir, mocker):
         f"[ -f setup.py ] || [ -f pyproject.toml ] && {new_dir}/parts/p1/install/bin/pip install  -U .",
         *get_build_commands(new_dir, should_remove_symlinks=True),
     ]
+
+
+def test_get_file_list(new_dir):
+    part_info = PartInfo(
+        project_info=ProjectInfo(application_name="test", cache_dir=new_dir),
+        part=Part("my-part", {}),
+    )
+    properties = PythonPlugin.properties_class.unmarshal({"source": "."})
+    plugin = PythonPlugin(properties=properties, part_info=part_info)
+
+    # Build a fake file tree, emulating what real package installs look like.
```

Comment:
I think we should do an actual file tree on a datadir for the test to use. It makes it easier to see the actual expected structure and review the actual test code (as opposed to code that is just generating test data)

Reply:
Like this? https://github.com/canonical/craft-parts/pull/943/commits/a3e158fcd49e595af744dca00670431aa2f67336